### ISSUE TITLE
istio-iptables.sh doesn't handle multiple excludes and include correctly

### ISSUE THREAD

**Issue by kramerul** [istio/istio#13979]
**Bug description**

If using the option `-i` or `-x` in istio-iptables.sh with a comma separated list of ip ranges, the script doesn't use the option at all.

For example the loop in line 229 

```
for range in "${OUTBOUND_IP_RANGES_EXCLUDE[@]}"; do
    r=${range%$pl}
```
runs only once, because `OUTBOUND_IP_RANGES_EXCLUDE` is no array variable. It's a simple string variable build from a command line argument.

**Expected behavior**

Script works correctly

**Steps to reproduce the bug**

**Version (include the output of `istioctl version --remote` and `kubectl version`)**

master

**How was Istio installed?**

N/A

**Environment where bug was observed (cloud vendor, OS, etc)**

N/A

**Affected product area (please put an X in all that apply)**

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[X] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[X] Security
[ ] Test and Release
[ ] User Experience

Additionally, please consider attaching a [cluster state archive](http://istio.io/help/bugs/#generating-a-cluster-state-archive) by attaching
the dump file to this issue.

**Comment by kramerul** [comment]
This is already fixed in #13579

**Comment by kramerul** [comment]
This is fixed by 8dd9e363eefef5dfaefe7661d271a91bc8e2d2f2
